spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/MappingUtils.java

```diff
@@ -32,6 +32,10 @@
  */
 public class MappingUtils {
 
+	private MappingUtils() {
+		super();
```

Comment:
Would you mind to explain why do we need call `super` from all of these new `private` ctors?
Looks like that works as is even without it and `checkstyle` is happy, too.